### PR TITLE
Added burst rate display support; refactored info printing to be pdns system specific

### PR DESCRIPTION
**Pull request by djw1149**

### CODE REVIEW

File: dnsdbq.c [hunk 1]
```diff
@@ -1314,10 +1316,33 @@ print_rateval(FILE *outstream, const char *key, const struct rateval *tp) {
...
+		return;
+
+	assert(tp_size->rk == rk_int);
+	assert(tp_window->rk == rk_int);
+
+	u_long b_w = tp_window->as_int;
+	u_long b_s = tp_size->as_int;
+
+	fprintf(outstream, "\t%s: ", key);
+	if (b_w == 3600) fprintf(outstream, "%lu per hour", b_s);
```
Comment (bapril) [inline dnsdbq.c]: Please correct the formatting here. the if or else shouldn't be on the same line as the statement that it is conditionalizing.

File: dnsdbq.c [hunk 2]
```diff
@@ -1314,10 +1316,33 @@ print_rateval(FILE *outstream, const char *key, const struct rateval *tp) {
 	fputc('\n', outstream);
 }
 
-/* write_info -- assumes that reader contains the complete json block
+/* print_burstrate -- output formatter for burst_size and burst_window ratevals
  */
 static void
-write_info(reader_t reader) {
+print_burstrate(FILE *outstream, const char *key, const struct rateval *tp_size, const struct rateval *tp_window) {
```
Comment (bapril) [inline dnsdbq.c]: Let's also keep 80 cols in mind.

File: dnsdbq.c (hunk 1 above)
Comment (djw1149) [reply dnsdbq.c]: Fixed.

File: dnsdbq.c (hunk 2 above)
Comment (djw1149) [reply dnsdbq.c]: Fixed.

**Comment by djw1149** [comment]
Please squash & merge with commit message: 'Extract burst_size, burst_window, and results_max from rate info and print it; refactored info printing to be pdns system specific'

**Pull request merged**